doc/cla/corporate/quartile.md

```diff
@@ -12,7 +12,6 @@ List of contributors:
 
 Yoshi Tashiro tashiro@quartile.co https://github.com/yostashiro
 Ryoko Tsuda ryoko@quartile.co https://github.com/Ryoko04
-Ai Kakurai ai@quartile.co https://github.com/kakurai8
```

Comment:
Why some people disappear from this list?
This should probably be in some other PR.